helper/populate_db.py

```diff
@@ -81,21 +85,50 @@ def populate_db_with_test_data(session):
         UserInterventionState(users_nicedayuid=40121, intervention_phase_id=1, intervention_component_id=5,
                               completed=False, last_time=datetime.now().astimezone(tz_nl), last_part=1),
         UserPreferences(users_nicedayuid=40121, intervention_component_id=5,
-                        recursive=True, week_days='1,2,3,4,5,6,7', preferred_time=time(17, 00, 00))
+                        recursive=True, week_days='1,2,3,4,5,6,7', preferred_time=time(14, 21, 00)),
+        UserPreferences(users_nicedayuid=40121, intervention_component_id=7,
+                        recursive=True, week_days='1,2,3,4,5,6,7', preferred_time=time(14, 22, 00)),
+        UserPreferences(users_nicedayuid=40121, intervention_component_id=8,
+                        recursive=True, week_days='1,2,3,4,5,6,7', preferred_time=time(14, 23, 00)),
+        UserPreferences(users_nicedayuid=40121, intervention_component_id=9,
+                        recursive=True, week_days='1,2,3,4,5,6,7', preferred_time=time(14, 24, 00)),
+        UserPreferences(users_nicedayuid=40121, intervention_component_id=10,
+                        recursive=True, week_days='1,2,3,4,5,6,7', preferred_time=time(14, 25, 00))
     ]
     [session.merge(obj) for obj in objects]
 
     session.commit()
 
 
-def initialize_intervention_components_table():
+def initialize_preparation_components_table():
+    data = [
```

Comment:
OK, so these are not per user, right? Just configuring stuff the scheduler needs to know about?

Reply:
Yes, exactly, this is not per user, but populates the tables with fixed information